apps/admin/frontend/src/api.ts

```diff
@@ -233,6 +233,18 @@ export const getPrintedBallots = {
   },
 } as const;
 
+export const getCurrentSystemSettings = {
```

Comment:
Nit that might be related to my above comment re not needing to persist multiple system settings objects: Could shorten this to `getSystemSettings`